includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -43,12 +62,31 @@ public function register() {
 	protected function enqueue_gtag_script() {
 		if ( did_action( 'googlesitekit_analytics_init_tag' ) ) {
 			// If the gtag script is already registered in the Analytics module, then we need to add <MEASUREMENT_ID> configuration only.
-			$config = sprintf( 'gtag("config", "%s");', esc_js( $this->tag_id ) );
-			wp_add_inline_script( 'google_gtagjs', $config );
+			$this->add_inline_config( $this->tag_id, array() );
```

Comment:
This looks good in terms of retaining the current logic. However there is this point in the IB:

> Ensure this method is called consistently when both UA and GA4 are connected, or if only GA4 is connected. (Currently, if UA is enabled, then we do not add config options and do not call the filter correctly within `Analytics_4/Web_Tag::enqueue_gtag_script()`).

I must admit, I find this point a little obscure. To save me going down a rabbit hole with it, can you confirm whether you've tackled this point and what the resolution is if so?

Reply:
I have adjusted logic a little bit to make sure that we add custom dimensions only to the GA4 measurement ID config.